src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,96 @@
+/* = Navigation
+============================ */
+.tribe-common {
+
+	.tribe-common-c-nav {
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+
+		ul {
+			display: block;
+			list-style: none;
+			width: 100%;
+		}
+	}
+
+	.tribe-common-c-nav__prev,
+	.tribe-common-c-nav__next {
+		display: inline-flex;
+		align-items: center;
+		color: var(--color-text-secondary);
+		font-size: var(--font-size-1);
+		font-weight: var(--font-weight-bold);
+
+		@media (--viewport-medium) {
+			font-size: var(--font-size-2)
+		}
+
+		&:hover {
+			color: var(--color-navigation-hover);
+		}
+
+		&:disabled {
+			color: var(--color-text-disabled);
+		}
+
+		&:before,
+		&:after {
+			background-repeat: no-repeat;
+			background-size: contain;
+			content: '';
+			height: 14px;
+			width: 12px;
+		}
+	}
+
+	.tribe-common-c-nav__prev {
+		float: left;
+
+		&:before {
+			background-image: svg-inline(caret-left);
+			margin-right: 10px;
+		}
+
+		&:active,
+		&:hover,
+		&:focus {
+
+			&:before {
+				background-image: svg-inline(caret-left-active);
+			}
+		}
+
+		&:disabled {
+
+			&:before {
+				background-image: svg-inline(caret-left-disabled);
+			}
+		}
+
+	}
+
+	.tribe-common-c-nav__next {
+		float: right;
```

Comment:
Let's not use floats here, otherwise the parent elements of `.tribe-common-c-nav__prev` and `next` now have no children, and their sizes collapse, resulting in a navigation that has 0 height and 0 width. This makes it difficult to build markup and style around.